code/game/objects/items/cosmetics.dm

```diff
@@ -289,6 +289,76 @@
 		sharpness = IS_BLUNT
 		tool_behaviour = null
 
+/obj/item/comb
+	name = "comb"
+	desc = "A slick pocket comb, for cool people only."
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "comb"
+	w_class = WEIGHT_CLASS_SMALL
+	force = 2
+	throwforce = 2
+	throw_speed = 6  // like card throwing
+	throw_range = 10
+	var/applied_hair = "Combover"  // incase anyone wants to make a bedhead-inator or something
+
+/obj/item/comb/suicide_act(mob/user)
+	user.visible_message("<span class='suicide'>[user] is trying to sweep their hair so fast it combusts! It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	var/mob/living/carbon/human/departed = user
+	departed?.hair_style = "Balding Hair"  // oh no it burned off!
+	departed.update_hair()
+	return (FIRELOSS)
+
+
+/obj/item/comb/attack(mob/living/M, mob/living/user)
+	var/mob/living/carbon/human/patient = M
+	var/mob/living/carbon/human/barber = user
+
+	if(patient == barber)
+		return attack_self(patient)
+	if(barber.a_intent == INTENT_HARM )
+		return ..()
+	if(!hair_check(M, user))
+		return
+
+	barber.visible_message("<span class='notice'>You attempt to sweep [patient]'s hair.</span>", "<span class='notice'>[barber] reaches towards [patient]'s hair with a comb.</span>")
+	if(!do_after(user, 3, target=patient))
+		return
+	patient.hair_style = applied_hair
+	patient.update_hair()
+	patient.visible_message("<span class='notice'>[user] carefully sweeps [patient]'s hair.</span>", "<span class='notice'>[user] carefully reaches forward and sweeps your hair!</span>")
+
+
+/obj/item/comb/attack_self(mob/user)
+	. = ..()
+	if(!hair_check(user, user))
+		return
+
+	var/mob/living/carbon/human/coolguy64 = user
+
+	coolguy64.hair_style = applied_hair
+	coolguy64.update_hair()
+	coolguy64.visible_message("<span class='notice'>[coolguy64] sweeps their hair back.</span>", "<span class='notice'>You sweep your hair back.</span>")
```

Comment:
Variable naming

Reply:
how is this a varname issue its a cool guy cant i have little jokes in my code?!